meta-ivi/recipes-extended/dlt-daemon/dlt-daemon_2.18.1.bb

```diff
@@ -12,26 +12,25 @@ SECTION = "console/utils"
 LICENSE = "MPLv2"
 LIC_FILES_CHKSUM = "file://LICENSE;md5=8184208060df880fe3137b93eb88aeea"
 
-PR = "r2"
-
 DEPENDS = "dbus zlib gzip-native"
 
-SRCREV = "bdde2374efd567f27d7a4ec70911b34cda061d27"
-SRC_URI = "git://github.com/GENIVI/${BPN}.git;protocol=https \
+SRC_URI = "https://github.com/GENIVI/dlt-daemon/archive/v${PV}.tar.gz \
     file://0002-Don-t-execute-processes-as-a-specific-user.patch \
     file://0004-Modify-systemd-config-directory.patch \
-    file://0001-Fix-compilation-with-glibc-2.28.patch \
-    "
-S = "${WORKDIR}/git"
+"
+SRC_URI[md5sum] = "6f2f6d3386991ed5d5e5078647c62ce5"
+SRC_URI[sha256sum] = "07f7a65d3d248e8f5eed820cae0e2ec81318417a81540647346f81cfe024ee65"
```

Comment:
I personally think double checksums are a bad practice because it only causes more maintenance effort.  Again, we don’t have a decided policy however.  (I’m not sure if OE/Yocto ecosystem has agreed on a best practice, but I hope it is a single checksum?)

Reply:
Virtually every recipe I have seen has had both md5 and sha256 checksum set, so this seems to be a common practice.

Reply:
Yes, very common. Unclear to me why though. Could not find a reason in https://www.yoctoproject.org/docs/latest/mega-manual/mega-manual.html .

Reply:
> I’m not sure if OE/Yocto ecosystem has agreed on a best practice, but I hope it is a single checksum?

Looking only at poky (@9dfebdaf7a):
```bash
$ grep -rI SRC_URI.*md5sum . | wc -l
510
$ grep -rI SRC_URI.*256sum . | wc -l
508
$ grep -rI -A 1 SRC_URI.*md5sum . | grep 256sum | wc -l
506
```

Seems like they think best practice is double.